tests/cases/warden_precompile_action.go

```diff
@@ -226,7 +226,8 @@ func (c *Test_WardenPrecompileAction) Run(t *testing.T, ctx context.Context, bui
 			2,
 			0,
 			"any(1, warden.space.owners)",
-			"any(1, warden.space.owners)")
+			"any(1, warden.space.owners)",
+			uint8(types.BroadcastType_BROADCAST_TYPE_DISABLED))
```

Comment:
You have uint8 here and int32 below, looks like they should match

Reply:
please read resolved comments above